api/routes/analysis.py

```diff
@@ -241,8 +247,9 @@ async def query_analyses(
 @router.get('/analysis-runner', operation_id='getAnalysisRunnerLog')
 async def get_analysis_runner_log(
     project_names: list[str] = Query(None),  # type: ignore
-    author: str = None,
+    # author: str = None, # not implemented yet, uncomment when we do
```

Comment:
This feels like you've merged something bad?

Reply:
Wow something definitely went wrong in that merge commit, did not realise how mangled this branch got. I'll have to look at my terminal history to see what exactly happened. Sorry bout this, I'll fixerup in the morning 🪠